### PR TITLE
adb*, chmod, weasyprint: add Russian translation

### PR DESCRIPTION
**Pull request by Bodyhealer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ru/common/adb-reverse.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# adb reverse
+
+> Android Debug Bridge Reverse: обратные подключения сокета из экземпляра эмулятора Андроид или подключённого Андроид устройства.
+> Больше информации на <https://developer.android.com/studio/command-line/adb>.
```
Comment (navarroaxel) [inline pages.ru/common/adb-reverse.md:4]:
```suggestion
> Больше информации: <https://developer.android.com/studio/command-line/adb>.
```

File: pages.ru/common/adb-install.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# adb install
+
+> Android Debug Bridge Install: Передать пакеты на экземпляр эмулятора Андроид или подключённое Андроид устройства.
+> Больше информации на <https://developer.android.com/studio/command-line/adb>.
```
Comment (navarroaxel) [inline pages.ru/common/adb-install.md:4]:
```suggestion
> Больше информации: <https://developer.android.com/studio/command-line/adb>.
```

File: pages.ru/common/adb-shell.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# adb shell
+
+> Android Debug Bridge Shell: Запуск удалённых команд оболочки на экземпляре эмулятора Андроид или на подключённом Андроид устройстве.
+> Больше информации на <https://developer.android.com/studio/command-line/adb>.
```
Comment (navarroaxel) [inline pages.ru/common/adb-shell.md:4]:
```suggestion
> Больше информации: <https://developer.android.com/studio/command-line/adb>.
```

File: pages.ru/common/weasyprint.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# weasyprint
+
+> Переводить HTML в PDF или PNG.
+> Больше инофрмации: <https://weasyprint.org/>.
```
Comment (navarroaxel) [inline pages.ru/common/weasyprint.md:4]: A typo I guess:
```suggestion
> Больше информации: <https://weasyprint.org/>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @Bodyhealer, I don't speak the language but I have a few comments.

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Review by schneiderl — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**